From 34be36a62ff0b3c701a965bdd129662eacae5388 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@nxp.com>
Date: Tue, 24 Jul 2018 16:39:08 +0800
Subject: [PATCH 4223/5242] MLK-19029 net: can: flexcan: fix the build warning
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

commit  fcab16038dd393ded58fa96e67fbc903bb9aaff3 from
https://source.codeaurora.org/external/imx/linux-imx.git

linux-imx/drivers/net/can/flexcan.c: In function ‘flexcan_probe’:
linux-imx/drivers/net/can/flexcan.c:1506:2: warning: label
‘failed_register’ defined but not used [-Wunused-label]
  failed_register:
    ^~~~~~~~~~~~~~~

Reviewed-by: Dong Aisheng <aisheng.dong@nxp.com>
Signed-off-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/net/can/flexcan.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/net/can/flexcan.c b/drivers/net/can/flexcan.c
index ab90d42..644aca0 100644
--- a/drivers/net/can/flexcan.c
+++ b/drivers/net/can/flexcan.c
@@ -1553,7 +1553,6 @@ static int flexcan_probe(struct platform_device *pdev)
  failed_rpm_disable:
 	pm_runtime_disable(&pdev->dev);
  failed_offload:
- failed_register:
 	free_candev(dev);
 
 	return err;
-- 
1.7.9.5

